### DISCUSSION TITLE
Support for RSS in general?

### DISCUSSION THREAD

**Discussion by spacecowboy** [OpenPodcastAPI/api-specs#68] [General / Meta]
Hi, just attended your talk at FOSDEM and I wonder if you have considered supporting RSS feeds in general, not specifically podcasts.

RSS feeds for text/news faces the same problems as podcasts with syncing subscriptions, read status, discovering new feeds, etc.

Mostly what makes a podcast different is the enclosure, and associated playback position.

So have you ever considered it? 
Would you be willing to consider it?

For context I am the dev behind the RSS reader Feeder: https://github.com/spacecowboy/feeder

**Comment by georgkrause** [comment]
Hello @spacecowboy and sorry for the late reply!

Everyone here is somehow invested in Podcasts, but we didn't talked about RSS specifically. Some things are really specific to Podcasts, like the playback position, which doesn't make a lot of sense for regular RSS entries. It would be interesting to consider how many changes would be required to support regular RSS feeds. I am not sure though if we have the expertise or time to widen the scope.
